helix-term/src/commands.rs

```diff
@@ -270,6 +270,7 @@ impl MappableCommand {
         append_mode, "Insert after selection (append)",
         command_mode, "Enter command mode",
         file_picker, "Open file picker",
+        file_picker_in_directory_of_active_buffer, "Open file picker in the directory of the active buffer",
```

Comment:
It would be better if the description was shorter so that the infobox wouldn't be too big.

Reply:
Same for the command name, maybe `file_picker_in_active_buffer_dir`?